nncf/experimental/onnx/dataloaders/imagenet_dataloader.py

```diff
@@ -35,14 +35,19 @@ def __len__(self):
         return len(self.dataset)
 
 
-def create_dataloader_from_imagenet_torch_dataset(dataset_dir, input_shape: List[int], batch_size: int = 1,
```

Comment:
We should use dataset_dir directly as a path for ImageNetDataLoader, without adding hard-coded parts like a `train`.
For example, `torchvision.datasets.ImageFolder(dataset_dir, transform)`

Reply:
Done